web/controllers/post_controller.ex

```diff
@@ -5,20 +5,21 @@ defmodule ElixirTw.PostController do
 
   def index(conn, _params) do
     posts = fetch_posts
-
+            |> Enum.map( &update_comments_count/1 )
```

Comment:
There is no whitespace around parentheses/brackets most of the time, but here there is.